subprojects/platform-play/src/integTest/groovy/org/gradle/play/integtest/PlayPlatformIntegrationTest.groovy

```diff
@@ -56,6 +56,9 @@ model {
     }
 }
 """
+        if(playVersion.startsWith('2.2')){
```

Comment:
formatting (space after `if` and betweeen `)` & `{`)

Reply:
Nice catch, thanks!